app/databaseAccessors/articleAccessor.js

```diff
@@ -307,4 +307,27 @@ export default class ArticlesAccessor {
       ErrorInternalAPIModelFieldValidation(e);
     }
   }
+
+    /**
+   * updateArticle method
+   *
+   * updates the article with the parameter provided
+   *
+   * @param {string} slug
+   * @param {JSON} update
+   * @returns updated article
+   */
+    static async updateArticle(slug, update) {
+      try {
+        await Connection.open();
+        const article = await Article.findOneAndUpdate({ slug }, update, { new: true });
+        if (!article) {
+          throw new ErrorInternalAPIModelFieldValidation("Article not found");
+        }
+        return article;
+      } catch (e) {
+        console.error(e);
+        throw e;
```

Comment:
`throw e` is unhelpful here, since we don't really know what type of error we're getting out of this method when used.

Consider catching the error as either a ErrorDatabaseConnection/ErrorInternalDatabaseAccessor/ErrorInternalUnexpected so that we know the particular error that happened when it does.